### DISCUSSION TITLE
怎么设置Andorid端的博客背景

### DISCUSSION THREAD

**Discussion by DSTBP** [YunYouJun/valaxy#435] [Q&A]
请问一下怎么设置Andorid端的博客背景图片，或者怎么将PC端的背景图片适配Android端，我的博客背景图片在Android端上在滑动过程中会时不时缩放 dstbp.com

**Comment by YunYouJun** [comment]
有点困惑，能否提供缩放的录屏？

**Reply by DSTBP** [reply]
https://github.com/user-attachments/assets/16fc24a8-6cdc-4fc7-a4b6-fec7eacc6d62

每次缩放都会顿一下，还有请问一下，怎么样能在博客全加载完全后在显示，在视频中是显现后再加载图片，能不能等图片加载完后再展示。图片不大，180kb左右。
